libs/sdk-ui-dashboard/src/presentation/layout/DefaultDashboardLayoutRenderer/interfaces.ts

```diff
@@ -19,7 +20,7 @@ export type IDashboardLayoutSectionKeyGetterProps<TWidget = IDashboardWidget> =
     /**
      * Current screen type with respect to the set breakpoints.
      */
-    screen: ScreenSize;
+    screen: ScreenClass;
```

Comment:
I am not sure if we can change this type of public interface, before it used our internal type. 
Should not we do some conversion from 
ScreenSize from "@gooddata/sdk-model"
to 
import { ScreenClass } from "react-grid-system";
internally?